uber/models.py

```diff
@@ -302,12 +302,12 @@ def _label(self, name, val):
         try:
             val = int(val)
         except ValueError:
-            log.debug('{} is not an int, did we forget to migrate data for {} during a DB migration?').format(val, name)
+            log.debug('{} is not an int, did we forget to migrate data for {} during a DB migration?'.format(val, name))
```

Comment:
Ha, whoops!  BTW, there's no need to even say ``.format`` in this case; we could write this line as simply

```python
log.debug('{} is not an int, did we forget to migrate data for {} during a DB migration?', val, name)
```